ad5758-sdz/ad5758.c

```diff
@@ -286,7 +286,7 @@ int32_t ad5758_set_dc_dc_conv_mode(struct ad5758_dev *dev,
 	 * mode.

```

Comment:
General remark: the Signed-off-by should include your full name: Stanca [space] Pop